test/SignalRServiceExtension.Tests/AzureSignalRClientTests.cs

```diff
@@ -38,7 +38,8 @@ public async Task GetClientConnectionInfo()
             var configDict = new Dictionary<string, string>() { { Constants.ServiceTransportTypeName, "Transient" }, { connectionStringKey, connectionString } };
             var configuration = new ConfigurationBuilder().AddInMemoryCollection(configDict).Build();
             var serviceManagerStore = new ServiceManagerStore(configuration, NullLoggerFactory.Instance, new TestRouter());
-            var azureSignalRClient = new AzureSignalRClient(serviceManagerStore, connectionStringKey, hubName);
+            StaticServiceHubContextStore.ServiceManagerStore = serviceManagerStore;
+            var azureSignalRClient = await Microsoft.Azure.WebJobs.Extensions.SignalRService.Utils.GetAzureSignalRClient(connectionStringKey, hubName);
```

Comment:
>Microsoft.Azure.WebJobs.Extensions.SignalRService [](start = 43, length = 49)

too long, put it as `using ...`

Reply:
There is another  `Utils`, so we can not use `using ...Utils.GetAzureSignalRClient(...)` https://github.com/Azure/azure-functions-signalrservice-extension/blob/48f279e29f1d9234b95e59ea019e4519af95775f/test/SignalRServiceExtension.Tests/AzureSignalRClientTests.cs#L17

Reply:
using SignalRUtils = ....

---
In reply to: [603017311](https://github.com/Azure/azure-functions-signalrservice-extension/pull/193#discussion_r603017311) [](ancestors = 603017311)